packages/common/lib/find-content-by-id.ts

```diff
@@ -1,9 +1,4 @@
-import { FullCourseContent } from '@/db/course';
-
-export default function findContentById(
-  contents: FullCourseContent[],
-  ids: number[],
-) {
+export default function findContentById(contents: any[], ids: number[]) {
```

Comment:
why any here it was FullCourseContent?

Reply:
yes, we can't import from next-frontend as it will cause cyclic dependency